Parsers/ASimAuditEvent/Parsers/vimAuditEventCiscoISE.yaml

```diff
@@ -0,0 +1,359 @@
+        (array_length(srcipaddr_has_any_prefix) == 0 or has_any_ipv4_prefix(SyslogMessage, srcipaddr_has_any_prefix))
+        and (array_length(actorusername_has_any) == 0 or SyslogMessage has_any (actorusername_has_any))
+        and (array_length(operation_has_any) == 0 or SyslogMessage has_any (operation_has_any))
+        and (array_length(newvalue_has_any) == 0 or SyslogMessage has_any (newvalue_has_any))
+    | project
+        TimeGenerated,
+        EventTime,
+        EventOriginalType,
+        Computer,
+        HostName,
+        HostIP,
+        SyslogMessage
+    | lookup EventFieldsLookup on EventOriginalType
+    | parse-kv SyslogMessage as (NetworkDeviceName: string, ['User-Name']: string, UserName: string, User: string, ['Remote-Address']: string, ['Device IP Address']: string) with (pair_delimiter=',', kv_delimiter='=')
+    | project-rename
+        SrcIpAddr=['Remote-Address']
+        , TargetIpAddr =['Device IP Address']
+    | where (array_length(srcipaddr_has_any_prefix) == 0 or has_any_ipv4_prefix(SrcIpAddr, srcipaddr_has_any_prefix))
+    | extend DvcHostname = coalesce(NetworkDeviceName, Computer, HostName)
+    | extend ActorUsername = coalesce(['User-Name'], UserName, User)
+    | extend ActorUsernameType = _ASIM_GetUsernameType(ActorUsername)      
+    | where (array_length(actorusername_has_any) == 0 or ActorUsername has_any (actorusername_has_any))
+    | extend 
+        DvcIpAddr = iif(isnotempty(HostIP) and HostIP != "Unknown IP", HostIP, extract(@"(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})", 1, Computer)) 
+        , EventStartTime = coalesce(EventTime, TimeGenerated)
+        , EventEndTime = coalesce(EventTime, TimeGenerated)
+        , EventVendor = "Cisco"
+        , EventProduct = "ISE"
+        , EventProductVersion = "3.2"
+        , EventCount = int(1)
+        , EventSchema = "AuditEvent"
+        , EventSchemaVersion = "0.1.0"
+        , ObjectType = "Configuration Atom"
+        , TargetAppName = "ISE"
+        , TargetAppType = "Service"
+    // ***************** <Aliases> ********************
+    | extend 
+        Dvc = coalesce(DvcIpAddr, DvcHostname)
+        , Application = TargetAppName
+        , IpAddr = coalesce(SrcIpAddr, TargetIpAddr)
+        , Dst = TargetIpAddr
+        , Src = SrcIpAddr
+        , User = ActorUsername
+    // ***************** </Aliases> *******************
+    | project-away
+        EventTime,
+        Computer,
+        HostName,
+        SyslogMessage,
+        NetworkDeviceName,
+        ['User-Name'],
+        UserName
+    };
+    CiscoISEAuditParser(
+        starttime = starttime,
+        endtime = endtime,
+        srcipaddr_has_any_prefix = srcipaddr_has_any_prefix,
+        actorusername_has_any = actorusername_has_any,
+        eventtype_in = eventtype_in,
+        eventresult = eventresult,
+        operation_has_any = operation_has_any,
+        object_has_any=object_has_any,
+        newvalue_has_any=newvalue_has_any,
+        disabled=disabled
+    )
```

Comment:
Please re-test filters I found issues with the following filters:
eventtype_in not working
eventresult

Reply:
fixed